### DISCUSSION TITLE
BladeBit with Plotman

### DISCUSSION THREAD

**Discussion by brianwfreeman** [ericaltendorf/plotman#938] [Q&A]
Right now I have 64-core systems pumping out plots in 5-6 minutes using BladeBit; the copy to HDD part takes 12 minutes, so I have a bottle-neck there. I was thinking of configuring PlotMan to have the plots sent from RAM to a high speed SSD, and then let PlotMan handle the rsync transfer of those plots to the HDDs.

Has anyone done this successfully? Are you willing to share your plotman.yaml config file with us to see how you set it up?

**Comment by altendky** [comment]
That's just `dst` on one or more SSDs and then a local rsync archive.  But...  archive doesn't presently support multiple archive jobs so you still end up piling up plots.  There's interest in archive supporting multiple parallel jobs but no work has been done on it.

**Reply by brianwfreeman** [reply]
> That's just `dst` on one or more SSDs and then a local rsync archive. But... archive doesn't presently support multiple archive jobs so you still end up piling up plots. There's interest in archive supporting multiple parallel jobs but no work has been done on it.

Thank you for your reply on this.  Does the PlotMan archive feature currently run at the same time as the plotting job?  Basically, will it do its best to move final plots from the destination drive to the archive drive during active plotting, but it simply won't be able to keep up with my speed of cranking out new plots?

**Reply by altendky** [reply]
Yes, they operate simultaneously.  In both cases plotman is just running a loop where every so often it decides if it should start a new plot process and also if it should start a new archive process.  Those processes run off and operate independently once launched.
